tutorials/noisepy_scedc_tutorial.ipynb

```diff
@@ -272,10 +275,21 @@
       "outputs": [],
       "source": [
         "#stations = \"RPV,STS,LTP,LGB,WLT,CPP,PDU,CLT,SVD,BBR\".split(\",\") # filter to these stations\n",
-        "stations = \"RPV,SVD,BBR\".split(\",\") # filter to these stations\n",
+        "# stations = \"RPV,SVD,BBR\".split(\",\") # filter to these stations\n",
         "# stations = \"DGR,DEV,DLA,DNR,FMP,HLL,LGU,LLS,MLS,PDU,PDR,RIN,RIO,RVR,SMS,BBR,CHN,MWC,RIO,BBS,RPV,ADO,DEV\".split(\",\") # filter to these stations\n",
+        "\n",
+        "# There are 2 ways to load stations: You can either pass a list of stations or load the stations from a text file.\n",
+        "\n",
+        "# ConfigParameters.load_stations(config,[\"RPV\",\"SVD\",\"BBR\"])\n",
+        "\n",
+        "# For loading it from a text file, write the path of the file in stations_file field of config instance as below\n",
+        "\n",
+        "# Now, relative_path contains the relative path to the \"stations.txt\" file\n",
+        "config.stations_file = os.path.abspath(os.path.join(os.getcwd(), '..', 'tests', 'data', 'stations.txt' ))\n",
```

Comment:
As mentioned above, I would leave this commented out as an example and I suggest just using a fake file name, like `my_stations.txt`. People who run these notebooks will not necessarily have cloned the repo so they may not have the `../tests/data/` directory.